statsmodels/stats/proportion.py

```diff
@@ -188,21 +193,24 @@ def _check(x: np.ndarray, name: str) -> np.ndarray:
         nobs_a = _check(np.asarray(nobs_a), "count")
 
     q_ = count_a / nobs_a
-    alpha_2 = 0.5 * alpha
+
+
+    if alternative == 'two-sided':
+        if method != "binom_test":
```

Comment:
Why not for binom_test?

Reply:
Because scipy.stats.binomtest uses "two-sided" as default for alternative, the original alpha should be used instead of  alpha / 2.0.
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.binomtest.html